recipes-ni/ni-safemode-utils/ni-safemode-utils.bb

```diff
@@ -1,33 +1,39 @@
 SUMMARY = "nilrt safemode utilities"
 DESCRIPTION = "nilrt distro-specific safemode utilities that provide basic system functionality."
+HOMEPAGE = "https://github.com/ni/meta-nilrt"
+SECTION = "base"
 LICENSE = "MIT"
 LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
-SECTION = "base"
+
+DEPENDS = "niacctbase"
```

Comment:
Why have the remaining packages been removed from this list?

Reply:
`shadow-native` and `pseudo-native` were bulk-added to a bunch of recipes many many years ago to fix some issue with how `pseudo` behaved. Since then, the issue has been resolved upstream and these `DEPENDS` are no longer needed. The `update-rc.d-native` DEPENDS is not needed because this recipe doesn't call `update-rc.d` anywhere.

I'll add those justifications to the commit message.